core/snapshots/storage/bolt.go

```diff
@@ -213,6 +213,37 @@ func GetSnapshot(ctx context.Context, key string) (s Snapshot, err error) {
 	return
 }
 
+// GetSnapshot returns the metadata for any status snapshot transaction
+// referenced by the given key. Requires a context with a storage transaction.
+func GetALLSnapshot(ctx context.Context, key string) (s Snapshot, err error) {
```

Comment:
```suggestion
func GetSnapshot(ctx context.Context, key string) (s Snapshot, err error) {
```
Mean to be `GetSnapshot`?